scripts/lochness_create_template.py

```diff
@@ -227,7 +230,6 @@ def create_keyring_template(keyring_loc: Path, args: object) -> None:
                 'HOST': 'mediaflux.researchsoftware.unimelb.edu.au',
                 'PORT': '443',
                 'TRANSPORT': 'https',
-                'TOKEN': '**TOKEN_delete_this_line_if_no_token**',
```

Comment:
Can I still delete `PASSWORD` line and use `TOKEN`?

Reply:
Here, the change is in the function that creates the template. lochness.mediaflux module should work as it has been working before. I not sure if we removed using `token` part though.